src/berry.c

```diff
@@ -80,7 +84,7 @@ const struct Berry gBerries[] =
     {
         .name = _("Chesto"),
         .firmness = BERRY_FIRMNESS_SUPER_HARD,
-        .color = BERRY_COLOR_PURPLE,
+        .color = ((OW_BERRY_COLORS == GEN_6_XY) ? BERRY_COLOR_PURPLE : BERRY_COLOR_BLUE),
```

Comment:
We don't need the outer parentheses for sure, and I think ternaries like these also don't use parentheses for the condition